ak_vendor/enums.py

```diff
@@ -163,14 +163,27 @@ class Meta:
 
 @choices
 class SubmissionStatusEnum:
-    """
-    Submission status enum
-    """
-
     class Meta:
+        # Statuses if app is uploading using URL
+        STORE_NOT_STARTED = [-10, "Not Started"]
```

Comment:
Make it positive starting with a prefix... free from 1000 onwards

Reply:
Done